ta/Makefile

```diff
@@ -28,6 +28,7 @@ TA_DIRS := create_fail_test \
 
 ifeq ($(CFG_SECURE_DATA_PATH),y)
 TA_DIRS += sdp_basic
+TA_DIRS += sdp_perf
```

Comment:
Is there no way to add SDP support to `ta/aes_perf`? I'm concerned about code duplication which is quite big here.